Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -397,19 +394,32 @@
                     }
                   },
                   {
-                    "columnMatch": "ProcessedTrend",
-                    "formatter": 9,
+                    "columnMatch": "MemoryUtilizationPercent",
+                    "formatter": 18,
                     "formatOptions": {
-                      "min": 0,
-                      "palette": "blue"
+                      "thresholdsOptions": "icons",

```

Comment:
Do we need a warning threshold too?

Reply:
Set as following for both CPU and Memory:
Warning: > 80%
Error: > 95%